test/testbot.js

```diff
@@ -0,0 +1,17 @@
+const Bot = require('scuttle-testbot')
```

Comment:
Gotta fix the license header :)

Reply:
Why is it needed per file?

Reply:
The code we're producing in this grant is under the conditions that it's open source, and to guarantee that our compliance with licenses is clear and legally correct, we should use https://reuse.software/ by the Free Software Foundation Europe that automatically checks that all source files in a project are correctly self-describing what their legal status is. The CI job checks that for us and turns red when there is a problem. (PS: let's aim for CI to always be green, even if you're just adding templates/drafts, the tests should pass).

Reply:
Oh good. I didn't know about that, that's clear, thanks

Reply:
added